backend/src/dal/sqlalchemy/entities/base.py

```diff
@@ -0,0 +1,16 @@
+# if is sqlalchemy...
```

Comment:
This comment is bad, do you think someone can understand clearly your intentions without reading the entire file?